### PR TITLE
Standardise other groups of files

### PR DESCRIPTION
**Pull request by Xorima**
# Description

Adds standard files for our other categories or repositories,

Once this is agreed I will get file managers running on these categories as well

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
```
Comment (tas50) [inline LICENSE:189]: This is actually meant to be this way. It's telling the person to fill in their own copyright

File: standardfiles/ide/.github/lock.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+daysUntilLock: 30
```
Comment (tas50) [inline standardfiles/ide/.github/lock.yml:2]: Do we really want to roll out locking. It causes enough e-mail spam that it tends to result in folks unfollowing repos which seems to be the last thing we want.

File: standardfiles/ide/LICENSE [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020 Sous-Chefs
```
Comment (tas50) [inline standardfiles/ide/LICENSE:189]: This needs to not be sous

File: standardfiles/orb/.github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @sous-chefs/orbs
```
Comment (tas50) [inline standardfiles/orb/.github/CODEOWNERS:1]: Is this user on every repo? It will need to be added for this to work.

Comment (Xorima) [reply standardfiles/orb/.github/CODEOWNERS:1]: Its a group and for all orbs repos (circleci) that should be the group anyway

File: standardfiles/ide/LICENSE (hunk 3 above)
Comment (Xorima) [reply standardfiles/ide/LICENSE:189]: Fair point, removed the file from management

File: LICENSE (hunk 1 above)
Comment (Xorima) [reply LICENSE:189]: Fair point, removed the file from management

File: standardfiles/ide/.github/lock.yml (hunk 2 above)
Comment (Xorima) [reply standardfiles/ide/.github/lock.yml:2]: I can remove it if we want to? 

I think most are on 30 days now anyway, but your call?

**Pull request merged**